dictionary/README.md

```diff
@@ -2,4 +2,6 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
-
+Restart Strategy:
+Chose transient because it should only restart if it exits abnormally
```

Comment:
# -2

I see your logic, but I feel that `permanent` actually expresses the intent better. This agent should _never_ shut down, so let's tell the world.